src/Common/src/CoreLib/System/MemoryExtensions.Fast.cs

```diff
@@ -188,7 +188,7 @@ public static int LastIndexOf(this ReadOnlySpan<char> span, ReadOnlySpan<char> v
 
             if (value.Length == 0)
             {
-                return 0;
+                return span.Length - 1;
```

Comment:
@ahsonkhan, @ViktorHofer, @jkotas, this behavior looks incorrect when `span` is empty, at least in that it doesn't match the behavior with strings, e.g. this:
```C#
using System;

class Program
{
    static void Main(string[] args) => Console.WriteLine("".LastIndexOf(""));
}
```
outputs `0`, not `-1`.

Reply:
Thanks for spotting. I shouldn't have rushed fixing the issue as I thought it might be the reason for the failing test in https://github.com/dotnet/corefx/pull/30840. Fix: https://github.com/dotnet/coreclr/pull/18826